### DISCUSSION TITLE
How should I perform a firmware update on the GOPRO11 Black engineering/test unit?

### DISCUSSION THREAD

**Discussion by 6MEMS** [gopro/labs#958] [Q&A]
I bought a Gopro11Black on a second-hand trading platform, but I found that I bought an engineering/testing machine, after I bought it back, I found that I could take photos, record videos, etc., but I couldn't connect it to my cell phone's app, "Gopro Quik", and when I connected it to my cell phone, it showed that I needed to update the firmware, and when I clicked on update, the Gopro would stay on the update screen (showing a blue gear), and it wouldn't complete the update for how long. After clicking update, the Gopro will always be in the update interface (displayed as a blue gear), and the update will not be completed for a long time, which makes me very annoyed!

How can I update this gopro to connect to "Gopro Quik" properly?

I've taken some pictures of the Gopro and put them below.

Thank you very much for your reply!

![image](https://github.com/gopro/labs/assets/149286029/05e3a8dd-75e0-424b-8e0e-eff376030e6c)

![image](https://github.com/gopro/labs/assets/149286029/efdb4440-6588-4aac-8c50-1166cb8bb09f)

![image](https://github.com/gopro/labs/assets/149286029/90509196-f7d9-4b2e-a813-8503acc05248)

![image](https://github.com/gopro/labs/assets/149286029/d3ae3c7b-2705-476d-b273-db6038e5ceb2)

**Comment by dnewman-gpsw** [comment]
Pre-production cameras use different firmware for security reasons, and therefore can't be updated in the field, as it not for end users.  You should not have have been able to purchase that camera.

**Reply by 6MEMS** [reply]
Thank you for your reply, I have bought this project/test machine and am unable to return it, which has caused me a lot of problems. Is there a repair or solution for this please, I just want it to connect to my phone to transfer photos properly now.

**Reply by dnewman-gpsw** [reply]
Nothing I can do.  You will need to replace that camera.

**Reply by 6MEMS** [reply]
Ok thanks, I'll just have to use the card reader to take out the shots then.
